### PR TITLE
Misc cleanup

### PR DESCRIPTION
**Pull request by lprichar**
Was just doing a final test and review.  Everything works beautifully, and in many cases even better than before.  Love it!  Here are just a few final things I cleaned up:

* Moved actions columns back to right-side of tables.  Feel free to weigh in and disagree, but I think the final PR will have fewer deltas for maintainers to review and go smoother if we minimize unnecessary UI changes.
* Centered the vertical orientation of all content in table rows
* Removed the very last reference to the MaterialInput component in the search bar and deleted the component (yay)
* Removed a bunch of orphan javascript code related to adminbsp in admin.js (this makes me happy)
* Tidied up some text in Role dialog tab labels

I would additionally love to go through `bsb-theme\css\style.css` and remove orphan css code, I bet there's a ton.  Anyone have experience doing that?  That would really be the coup de grace of this PR if we could pull it off.

### CODE REVIEW

File: angular/src/app/roles/roles.component.html [hunk 1]
```diff
@@ -43,8 +43,10 @@ <h2>{{ "Roles" | localize }}</h2>
                       })
               "
             >
+              <td>{{ role.name }}</td>
+              <td>{{ role.displayName }}</td>
               <td>
-                <button mat-icon-button [matMenuTriggerFor]="actionsMenu">
+                <button class="action-button" mat-icon-button [matMenuTriggerFor]="actionsMenu">
```
Comment (lprichar) [inline angular/src/app/roles/roles.component.html]: I don't know if this is a particularly good class name or not, up for suggestions.  I noticed there's a `header-dropdown-mat-icon-button` above which is a lot more detailed, but I didn't know what a corresponding name would be

File: angular/src/app/tenants/tenants.component.html [hunk 2]
```diff
@@ -79,6 +64,21 @@ <h2>{{ "Tenants" | localize }}</h2>
                   indeterminate_check_box
                 </i>
               </td>
+              <td>
+                <button class="action-button" mat-icon-button [matMenuTriggerFor]="actionsMenu">
```
Comment (lprichar) [inline angular/src/app/tenants/tenants.component.html]: There's a lot of churn in this section and the one below, but the only real change is adding class="action-button" which I did to center the button vertically

File: angular/src/app/roles/roles.component.html (hunk 1 above)
Comment (iyilm4z) [reply angular/src/app/roles/roles.component.html]: English is not my native language that’s why you better chose names for us:)

File: angular/src/app/tenants/tenants.component.html (hunk 2 above)
Comment (iyilm4z) [reply angular/src/app/tenants/tenants.component.html]: 👌

**Comment by iyilm4z** [comment]
@lprichar thanks for all of your efforts!

File: angular/src/app/roles/roles.component.html (hunk 1 above)
Comment (lprichar) [reply angular/src/app/roles/roles.component.html]: :) I honestly wouldn't have known if you hadn't said so, your English is excellent

**Pull request merged**